guides/common/modules/proc_creating-an-installation-medium-for-ubuntu-22-04.adoc

```diff
@@ -45,5 +45,15 @@ Ensure the path in `/pub/` matches the path in your _Preseed default PXELinux Au
 # rm -f {iso_image}
 ----
 
++
+[NOTE]
+====
+As an alternative to copying the ISO image content, you can create a symbolic link at `/var/www/html/pub/installation_media/ubuntu/22.04-x86_64` that points to the mounted image.
+The ISO image will then remain mounted so that it is accessible to HTTP clients.
```

Comment:
Suggestion for wording with active voice to make it clearer that it's an action for the user.
```suggestion
Keep the ISO image mounted so that it is accessible to HTTP clients.
```